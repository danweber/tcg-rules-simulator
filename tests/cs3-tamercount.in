VERSION:2

# link rules check

FIELD1:Attack1,CS3-06 Attack2,CS3-06 Attack3,CS3-06
HAND1:CS3-Birdwatcher CS3-Birdwatcher
TRASH1:CS3-Birdwatcher
SECURITY2:DW1-Van DW1-Van DW1-Van
GAME:P1 T3 MAIN 6

DUMP:HAND,TRASH

ECHO: -- attack! play a thing
CMD:ATTACK Attack1 Player
STEP:AUTOANSWER:CAPTURE
DUMP:HAND,TRASH

ECHO: -- attack a second time, play a thing
CMD:ATTACK Attack2 Player
STEP:AUTOANSWER:CAPTURE
DUMP:HAND,TRASH

ECHO: -- attack a third time, play a thing
CMD:ATTACK Attack3 Player
STEP:AUTOANSWER:CAPTURE
DUMP:HAND,TRASH



EXIT


ECHO: -- evo into thing without link +3
ECHO: -- we should get to select the things we trash
ECHO: -- also the order things go into trash doesn't make sense

CMD:EVO CS3-11 Multi
DUMP:DP,KEYWORDS,HAND,TRASH

EXIT
