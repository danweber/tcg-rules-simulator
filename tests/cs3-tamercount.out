FIELD1:Attack1,CS3-Shieldwing Attack2,CS3-Shieldwing Attack3,CS3-Shieldwing
FIELD2:
TRASH1:CS3-17
TRASH2:
HAND1:CS3-17 CS3-17
HAND2:
GAME:P1 T3 MAIN 6
ECHO: -- attack! play a thing
CAPTURE:Activate ShieldWing playing
CAPTURE:Don't pay/activate
CAPTURE:Target Birdwatcher Bill HAND-CS3-17
CAPTURE:Target Birdwatcher Bill HAND-CS3-17
CAPTURE:Target Birdwatcher Bill TRASH-CS3-17
FIELD1:Attack1,CS3-Shieldwing,REST Attack2,CS3-Shieldwing Attack3,CS3-Shieldwing I4,CS3-17
FIELD2:
TRASH1:CS3-17
TRASH2:DW1-Vanillamon
HAND1:CS3-17
HAND2:
GAME:P1 T3 MAIN 6
ECHO: -- attack a second time, play a thing
CAPTURE:Activate ShieldWing playing
CAPTURE:Don't pay/activate
CAPTURE:Target Birdwatcher Bill HAND-CS3-17
CAPTURE:Target Birdwatcher Bill TRASH-CS3-17
FIELD1:Attack1,CS3-Shieldwing,REST Attack2,CS3-Shieldwing,REST Attack3,CS3-Shieldwing I4,CS3-17 I5,CS3-17
FIELD2:
TRASH1:CS3-17
TRASH2:DW1-Vanillamon DW1-Vanillamon
HAND1:
HAND2:
GAME:P1 T3 MAIN 6
ECHO: -- attack a third time, play a thing
FIELD1:Attack1,CS3-Shieldwing,REST Attack2,CS3-Shieldwing,REST Attack3,CS3-Shieldwing,REST I4,CS3-17 I5,CS3-17
FIELD2:
TRASH1:CS3-17
TRASH2:DW1-Vanillamon DW1-Vanillamon DW1-Vanillamon
HAND1:
HAND2:
GAME:P1 T3 MAIN 6
